domains/auth/Auth.API/Integration.Tests/Integration.Tests.csproj

```diff
@@ -43,4 +43,8 @@
         </None>
     </ItemGroup>
 
+    <ItemGroup>
+      <Compile Remove="IntegrationTestCollection.cs" />
```

Comment:
Why?

Reply:
Must be a lefover from my attempt to optimize Auth integration tests. I thought i got it all reverted.